podpac/core/coordinates/stacked_coordinates.py

```diff
@@ -58,7 +58,7 @@ class StackedCoordinates(BaseCoordinates):
         
     """
 
-    _coords = tl.Tuple(trait=tl.Instance(Coordinates1d), read_only=True)
+    _coords = tl.Tuple(trait=tl.Instance(Coordinates1d))
```

Comment:
You can change this underlying attribute to be an `OrderedDict` if it makes other things easier.

I think it should still be `read_only`, and you should use `set_trait('_coords', ...)` to modify it).

Reply:
The tuple is fine since the dims define which is which. If we want to make this an OrderedDict in the future for consistency or other reasons, then we can.

Switched it back to `read_only`